var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -195,6 +196,8 @@ class PyScipy(PythonPackage):
     # Intel OneAPI ifx claims to support -fvisibility, but this does not work.
     # Meson adds this flag for all Python extensions which include Fortran code.
     conflicts("%oneapi@:2023.0", when="@1.9:")
+    # Unknown build error, version ranges may be incorrect
+    conflicts("%oneapi@2024:", when="@:1.8")
```

Comment:
Let me know if you want to pick this one, or investigate the disappearing issue with `nrm` and `py-pip` conflict.

Reply:
I'll let @rgommers decide if we should keep this one and @manuelakuhn decide if we should keep the `nrm` conflict. I'm not in a rush to see this merged.

Reply:
I think it's fine to keep that conflict; I don't expect SciPy versions <=1.8 to work with OneAPI >=2024 without patching.